packages/cli/subtasks/pick-contract.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs');
+const { subtask } = require('hardhat/config');
+const inquirer = require('inquirer');
+const autocomplete = require('inquirer-list-search-prompt');
+const logger = require('@synthetixio/core-js/utils/logger');
+const { SUBTASK_PICK_CONTRACT } = require('../task-names');
+
+subtask(SUBTASK_PICK_CONTRACT, 'Pick contract to interact with').setAction(
+  async (taskArguments, hre) => {
+    logger.info('Pick a contract:');
+
+    const deploymentData = JSON.parse(fs.readFileSync(hre.deployer.paths.deployment));
```

Comment:
No need to parse the data, it's already parsed at `hre.deployer.deployment.[general|abis|sources]`